static/js/list_render.js

```diff
@@ -32,6 +32,9 @@ exports.create = function ($container, list, opts) {
         // the false flag here means "don't run `init`". This is because a
         // user is likely reinitializing and will have put .init() afterwards.
         // This happens when the same codepath is hit multiple times.
+        if (opts.parent_container) {
+            opts.parent_container.on("click", "[data-sort]", exports.handle_sort);
+        }
```

Comment:
Should this be put inside `_set_events()` instead?  That seems like the more logical way to implement this idea; putting it with the other click handle declaration.

Reply:
Will do. Thanks.